ogc/wms-layer-url/README.md

```diff
@@ -1,18 +1,33 @@
-# WMS Layer URL
+# WMS layer URL
```

Comment:
should we add brackets round the URL here like we have in other readme updates? I'm easy either way.